src/Microsoft.Identity.Web/WebApiExtensions/MicrosoftIdentityWebApiAuthenticationBuilderExtensions.cs

```diff
@@ -176,7 +176,8 @@ public static class MicrosoftIdentityWebApiAuthenticationBuilderExtensions
                     // Process CIAM tenants
```

Comment:
I think this does more than process CIAM, it can process any OIDC compliant authority.